app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,59 @@
+class PostsController < ApplicationController
+
+  def like
+    @post = Post.all.find(params[:id])
+      Like.create(user_id: current_user.id, post_id: @post.id)
```

Comment:
You can do it the way you have done it here, but you can do it shorter with:
`Like.create(user: current_user, post: @post)`
But you can also start with user or post:
`current_user.create_like(post: @post)`
`@post.create_like(user: current_user`